recipes/ptxcompiler/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "ptxcompiler" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/rapidsai/ptxcompiler/archive/refs/tags/{{ version }}.tar.gz
+    sha256: c7021a0946478404c4055a612fd725af95d53ec23eb3dd8d8bc0b5ac55eb44d0
+    folder: ptxcompiler
+
+build:
+  number: 0
+  skip: true  # [(not linux) or (cuda_compiler_version !=  "11.2")]
+  script_env:
+    - CPATH=/usr/local/cuda/include
+    - LIBRARY_PATH=/usr/local/cuda/lib64
+  script:
+    - cd ptxcompiler
+    - {{ PYTHON }} -m pip install . -vv
+  missing_dso_whitelist:
+    - '*libc.so*'        # [linux]
```

Comment:
I'm unconvinced that you absolutely need these. Typically this indicates a missing dependency